frontend/packages/ux-editor/src/components/FormComponent/FormComponent.module.css

```diff
@@ -50,7 +56,7 @@
   box-sizing: border-box;
 }
 
-.formComponentWithHandle:has(.dragHandle:hover) {
+.formComponentWithHandle:is(.dragHandle:hover) {
```

Comment:
`:is` er ikke det samme som `:has`, så dette vil ikke fungere her. Dette har for øvrig kun effekt på skyggen - hovedårsaken til problemet er nok `.buttons:has(button[aria-expanded='true'])` over.